examples/stm32/xively_demo_stm32f4xx_nucleo_wifi/Inc/user_data.h

```diff
@@ -0,0 +1,39 @@
+#ifndef __USER_DATA_H__
+#define __USER_DATA_H__
+
+#define USER_DATA_STR_LEN 64 /* Must be a multiple of sizeof(int32_t) for CRC periph */
+
+typedef struct
+{
+    int32_t crc_checksum; /* For internal use only - Set by user_data_save_to_flash() */
+
+    int32_t wifi_client_encryption_mode; /* Casted from WiFi_Priv_Mode */
```

Comment:
It is misleading. I'd be confused to read anything about the wifi in an ethernet project. Especially if there are two types: wifi and ethernet. I'd be confused where I am. But yeah, we may put there a comment and that's it. We have to handle this keeping the weight of this project in mind. So it is ok for now.